envi/memory.py

```diff
@@ -67,6 +67,25 @@ def __init__(self, arch=None):
         if arch is not None:
             self.setMemArchitecture(arch)
 
+        self.bigend = envi.ENDIAN_LSB
+
+    def getEndian(self):
```

Comment:
yes.  it was causing issues for certain MemoryObject subclasses (forgetting which off-hand)

Reply:
```vdb > stepi -V -C 100
0x020026e0:  rep: nop edx ; 0x4159b010 
0x020026e4:  repnz: jmp qword [rip + 63453] ; 'TraceEmulator' object has no attribute 'getEndian
```

Reply:
the TraceEmulator we use for hooking an emulator into VDB (which i've found more and more urgent and important in the last year, i have just been writing code around it instead of figuring out how to do it correctly because it's broken).